app/utils/push_notifications.js

```diff
@@ -79,29 +79,25 @@ const onPushNotification = async (deviceNotification) => {
 
     if (data.type === 'clear') {
         dispatch(markChannelAsRead(data.channel_id, null, false));
-    } else {
-        // get the posts for the channel as soon as possible
-        retryGetPostsAction(getPosts(data.channel_id), dispatch, getState);
```

Comment:
Is this logic moving somewhere or are we removing it on purpose?

Reply:
nice catch!! this is actually a mistake, I was testing some stuff!!